contracts/examples/OracleCreator.sol

```diff
@@ -0,0 +1,183 @@
+pragma solidity =0.6.6;
+
+import './../libraries/TransferHelper.sol';
+import './../libraries/DXswapOracleLibrary.sol';
+import './../libraries/DXswapLibrary.sol';
+import './../libraries/SafeMath.sol';
+
+contract OracleCreator {    
+    using FixedPoint for *;
+    using SafeMath for uint256;
+
+    event OracleCreated(
+        uint256 _oracleId,
+        address _pair,
+        uint256 _windowSize,
+        uint8 _granularity,
+        uint256 _periodSize,
+        uint256 _executionBountyWei
+    );
+
+    event OracleExecuted(
+        uint256 _oracleId
+    );
+
+    struct Oracle{
+      uint256 windowSize;
+      address pair;
+      uint8 granularity;
+      uint256 periodSize;
+      uint256 executionBountyWei;
```

Comment:
The bounty is per update, not execution, right? so this should be updateBountyWei